src/classes/TestResult.js

```diff
@@ -365,8 +365,9 @@ ${ this.error.stack }`);
 
 		if (this.tests) {
 			ret = new String(ret);
-			ret.children = this.tests.filter(t => t.stats.fail + t.stats.pending + t.stats.skipped > 0)
-				                     .flatMap(t => t.toString(o)).filter(Boolean);
+			ret.children = this.tests.flatMap(t => t.toString(o)).filter(Boolean);
```

Comment:
We probably still want to hide passed tests (with all passed descendants) unless we're in verbose mode.

Reply:
Fixed